src/explore.sh

```diff
@@ -160,6 +163,21 @@ function parse_explore_options()
         options_values['VERBOSE']=1
         shift
         ;;
+      --show-context | -C)
+        if [[ -n "${options_values['CONTEXT']}" ]]; then
+          options_values['ERROR']="Invalid arguments: Multiple context values!"
```

Comment:
There are other instances like line `150` where double quotes are used. Should I change them too?

Reply:
Hi @rodrigosiqueira , Double quotes are also being used throughout the switch case block to set the `options_values['ERROR']`
I think it would be a good idea to use single quotes in other cases too.